engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -97,7 +97,7 @@
     private LocalPlayer localPlayer;
     @In
     private WorldProvider worldProvider;
-    private Camera playerCamera;
+    public Camera playerCamera;
```

Comment:
Getting it from the Context is even better. In theory, having spoken with Immortius some time ago, we'd like to move away from annotation-based injection and use the context (and sub-contexts) more widely.

That been said:

```
@In 
private WorldRenderer worldRenderer;
```

doesn't work? It should.